docker-compose.production.yml

```diff
@@ -11,6 +11,10 @@ services:
       - GUNICORN_OPTS= --workers 50 --timeout 300  # 5 minutes before reload
       - OL_CONFIG=/olsystem/etc/openlibrary.yml
       - PYENV_VERSION=3.8.6
-      - USE_NGINX=true
     volumes:
       - ../olsystem:/olsystem
+    command: >
+      bash -c "ls -Fla /etc/nginx
+      && ln -sfv /olsystem/etc/nginx/nginx.conf /etc/nginx/nginx.conf
+      && ln -sfv /olsystem/etc/nginx/sites-available /etc/nginx/sites-available
```

Comment:
I am just spinning my wheels on this.  I can not `rm` or `chown` or `sudo` the destination files so I can not create the symlinks.  What else should I try?